src/backend/api/meals.js

```diff
@@ -1,12 +1,193 @@
-const express = require("express");
+const express = require('express');
 const router = express.Router();
-const knex = require("../database");
+const knex = require('../database');
+let errorTitle = '';
+let q = '';
+function mealsQueryBuilder(query) {
+  try {
+    errorTitle = '';
+    if (query.hasOwnProperty('availableReservations')) {
+      q = knex
+        .select(
+          'Meal.title',
```

Comment:
Always try and avoid capital letters! In programming they are pretty much only used when defining classes